Add DRF serializer field for jmodel.jDateTimeField (#139)
slashmili/django-jalali: README.rst

Django rest framework
---------------------

> There is a serializer fields corresponding to ``jmodels.JDateField`` and ``jmodels.JDateTimeField`` for DRF:

Reviewer: ```suggestion
There are serializer fields corresponding to ``jmodels.JDateField`` and ``jmodels.JDateTimeField`` for DRF:
```
